### PR TITLE
Add dashboard page

### PR DESCRIPTION
**Pull request by mi-yu**
Also includes changes merged from master

screenshot_4

### CODE REVIEW

File: packages/client/src/pages/FindAMentor/index.js [hunk 1]
```diff
@@ -70,6 +71,7 @@ const FindAMentorPage = () => {
     return (
         <FindAMentorWrapper>
             <FindAMentorSidebarWrapper>
+                <h2>Filter By</h2>
```
Comment (mi-yu) [inline packages/client/src/pages/FindAMentor/index.js:74]: still need to add the active filters, maybe will do in separate PR

**Comment by johancc** [comment]
Build is failing because the test requires Firestore to work. I'll get the test working then merge this branch.

**Comment by johancc** [comment]
I'll disable the test for now since its unrelated to the dashboard.

**Comment by johancc** [comment]
Test issues have been fixed and it has been synced with master, merging by tonight unless there are other changes I should wait for.
